interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -23,6 +24,11 @@ function __construct($template_mod = "general") {
       $this->assign("gbl_vitals_options",$GLOBALS['gbl_vitals_options']);
     }
 
+		function setFormId($form_id){
```

Comment:
I searched the code to see if the function was anywhere else and I did not find another instance of it.